### PR TITLE
Added preview capabilities

### PR DESCRIPTION
**Pull request by danieliser**
Using the data-preview attribute you can target another element that will be updated with the selected icon.

    <div class="dashicon-picker">
        <input id="icon_field" name="icon_field" type="hidden" value="dashicons-admin-generic">
        <span class="dashicons dashicons-star-filled" id="icon_preview"></span>
        <input class="button dashicons-picker" data-preview="#icon_preview" data-target="#icon_field" type="button" value="Choose Icon"/>
    </div>

### CODE REVIEW

File: js/dashicons-picker.js [hunk 1]
```diff
@@ -312,9 +315,6 @@
 					} else {
 						$( 'li:lt(25)', list ).appendTo( list );
 					}
-					preview
-						.prop('class', 'dashicons')
```
Comment (bradvin) [inline js/dashicons-picker.js]: A question - why do you set the class prop, and not do it in one addClass call?

Comment (danieliser) [inline js/dashicons-picker.js]: You could but you would need to dynamically remove the existing icon class as well. This way it's reset clean after the new icon is chosen each time.

Happy to test suggestions. Using as is now in a new WP plugin and works perfect in unison with a color picker on the same preview.

**Comment by bradvin** [comment]
thanks for contributing this

**Comment by danieliser** [comment]
Not a problem, currently using it in a plugin I just released today :+1:

**Pull request merged**